Uplift/Controllers/FilterViewController.swift

```diff
@@ -143,14 +147,22 @@ class FilterViewController: UIViewController, RangeSeekSliderDelegate {
         contentView.addSubview(startTimeClassTypeDivider)
 
         // CLASS TYPE SECTION
-        classTypeDropdown = UITableView(frame: .zero, style: .grouped)
+        classTypeDropdownHeader = DropdownHeaderView(frame: .zero)
+        contentView.addSubview(classTypeDropdownHeader)
+        
+        classTypeDropdownFooter = DropdownFooterView(frame: .zero)
+        classTypeDropdownFooter.clipsToBounds = true
+        contentView.addSubview(classTypeDropdownFooter)
+        
+        let classTypeGesture = UITapGestureRecognizer(target: self, action: #selector(self.dropHideClasses(sender:) ))
```

Comment:
feel free to give input but I think `hideClassesDropdown` is a bit clearer?

Reply:
it's the footer button that shows more or hides the classes